src/openms/source/DATASTRUCTURES/LPWrapper.cpp

```diff
@@ -742,21 +750,26 @@ namespace OpenMS
 
       // set maximum allowed CPU time before forced stop (dangerous!)
       //model.setDblParam(CbcModel::CbcMaximumSeconds,60.0*1);
-
-      // Do initial solve to continuous
-      model.initialSolve();
-
-
-      // solve
-      model.branchAndBound();
-      // if (verbose_level > 0) OPENMS_LOG_INFO << " Branch and cut took " << CoinCpuTime()-time1 << " seconds, "
-      //                                        << model.getNodeCount()<<" nodes with objective "
-      //                                        << model.getObjValue()
-      //                                        << (!model.status() ? " Finished" : " Not finished")
-      //                                        << std::endl;
-      for (Int i = 0; i < model_->numberColumns(); ++i)
-      {
-        solution_.push_back(model.solver()->getColSolution()[i]);
+      
+      try {
```

Comment:
not a good idea.
If an exception occurs noone will notice because the method returns a seemingly valid value

Reply:
Good point.  The `try/catch` blocks that were inserted are not needed.  I will remove them.